molecule/default/Dockerfile.j2

```diff
@@ -13,4 +13,5 @@ RUN apt-get update && \
 
 EXPOSE 8088/tcp
```

Comment:
All this ports should go under exposed_ports in molecule.yml file

Reply:
done